src/UserSpaceInstrumentation/UserSpaceInstrumentationTestLib.cpp

```diff
@@ -47,7 +47,7 @@ uint64_t ExitPayload() {
     if (skipped > 0) {
       printf(" ( %lu skipped events )\n", skipped);
     }
-    printf("Returned from function %#lx\n", current_return_address.function_address);
+    printf("Returned from function with id %#lx\n", current_return_address.function_id);
```

Comment:
`%u` instead of `%#lx`? I don't think we want to display the function ID as hex (for what it matters here).

Reply:
done, `%lu` is what printf expects for a unsigned long.

Reply:
Oh yeah, this is not `absl::StrFormat`, I'm getting too used to that.